filters/map_composition.py

```diff
@@ -144,8 +144,15 @@ def responseComplete(self):
                 style_manager = qgis_layer.styleManager()
                 style_manager.renameStyle('', 'default')
 
-                # Add layer to the registry
-                QgsMapLayerRegistry.instance().addMapLayer(qgis_layer)
+            # add basemap to the qgs project so it can be called later to create a thumbnail
+            qgis_layer = QgsRasterLayer('type=xyz&url=http://tile.osm.org/{z}/{x}/{y}.png?layers=osm', 'osm', 'wms')
+            if not qgis_layer.isValid():
+                print("osm not found")
```

Comment:
Hindari print sih kalau bisa

Reply:
Eh, avoid using `print`

Reply:
Use Logger, because we can't see it if it is using print statement.